spec/outputs/sns_spec.rb

```diff
@@ -2,17 +2,110 @@
 require "logstash/devutils/rspec/spec_helper"
 require 'logstash/outputs/sns'
 require 'logstash/event'
+require "logstash/plugin_mixins/aws_config"
+
+require "aws-sdk" # TODO: Why is this not automatically brought in by the aws_config plugin?
```

Comment:
Well, this let block for instance needs it since setup a mock on the Aws classes themselves:

```
  let(:instance) {
    allow(Aws::SNS::Client).to receive(:new).and_return(mock_client)
    inst = LogStash::Outputs::Sns.new
    allow(inst).to receive(:publish_boot_message_arn).and_return(nil)
    inst.register
    inst
  }
```

If you remove the require the tests will fail unfortunately.